radicle-node/src/service.rs

```diff
@@ -777,6 +799,29 @@ where
         // TODO
     }
 
+    fn disconnect_unresponsive_peers(&mut self, now: LocalTime) {
```

Comment:
nit: I think this could be borrowed, same as `keep_alive` below
```suggestion
    fn disconnect_unresponsive_peers(&mut self, now: &LocalTime) {
```